src/components/FavouritesStack.js

```diff
@@ -1,38 +1,20 @@
+import React, {Component} from 'react';
+import {StackNavigator} from 'react-navigation';
 
-import React, { Component } from 'react';
-import {
-  Platform,
-  StyleSheet,
-  Text,
-  View,
-  Dimensions,
-  Image,
-  CustomCallout,
-  Button,
-  AsyncStorage
-} from 'react-native';
-import MapView from 'react-native-maps';
-import { StackNavigator } from 'react-navigation';
-import { List, ListItem } from 'react-native-elements'
-import Swipeout from 'react-native-swipeout';
-import { NavigationActions } from 'react-navigation'
-
-import Header from 'mopsik_mobile/src/components/Header';
 import MopDetailsView from 'mopsik_mobile/src/components/MopDetailsView';
 import FavouriteMopsView from 'mopsik_mobile/src/components/FavouriteMopsView';
 
 
 MOPS = require('mopsik_mobile/src/config/mops');
-var _ = require('lodash');
-
+let _ = require('lodash');
```

Comment:
This line can be removed as lodash is not used in this file. It's a variable, but works like an import of library (https://lodash.com/).